pages/posts/2.tsx

```diff
@@ -0,0 +1,179 @@
+	if err != nil {
+		panic(err)
+	}
+	defer client.Close()
+
+	config, err := client.ServiceGetConfiguration(ctx,
+            &protocoltypes.ServiceGetConfiguration_Request{})
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Hello, world! My peer ID is", config.PeerID)
+}
+`}
+                  </code>
+                </pre>
+                <p>
+                  First we use
+                  <code className="language-js token string">context.WithCancel</code>
+                  to create a <code className="language-js token string">ctx</code>
+                  object which is used in the gRPC calls, and a<code className="language-js">cancel</code> function to call when the app exits. The Context type is part of the Go
+                  standard library and you can read
+                  <a href="https://pkg.go.dev/context"> its documentation </a> for more details.
+                </p>
+                <p>
+                  Next we call
+                  <code className="language-js token string"> weshnet.NewInMemoryServiceClient </code>
+                  to initialize the Wesh service and create a gRPC
+                  <code className="language-js token string"> client </code> that we use to interact with it. Wesh has many init functions for different configurations. A Wesh service
+                  can run in memory with a direct connection, or as a separate process with communication through an open socket. We will discuss these options in future blog
+                  posts. In this example, we initialize a Wesh service which runs in application memory with a direct connection, and creates a new in-memory user account and data
+                  store. If your app doesn’t export the data, then it is lost when the app exits. (It’s also possible to create a persistent data store, which we will explore
+                  later.)
+                </p>
+                <p>
+                  Finally, we interact with the Wesh service by calling
+                  <code className="language-js token string"> client.ServiceGetConfiguration </code> , passing the <code className="language-js token string"> ctx </code> that we created.
+                  This method gets the current configuration of the service and is{" "}
+                  <a href="https://buf.build/berty/weshnet/docs/main:weshnet.protocol.v1#weshnet.protocol.v1.ProtocolService.ServiceGetConfiguration">documented here</a>, along
+                  with all the other API calls. Each API call takes a Protobuf request message. In this case there are no parameters, so we create an empty
+                  <code className="language-js token string"> protocoltypes.ServiceGetConfiguration_Request </code> . (We will see more complex examples later.) And each API call
+                  returns a Protobuf reply message, or sometimes a stream object. In this case it is a ServiceGetConfiguration.Reply . So to finish, we print a happy message with
```

Comment:
Please link ServiceGetConfiguration.Reply to https://buf.build/berty/weshnet/docs/main:weshnet.protocol.v1#weshnet.protocol.v1.ServiceGetConfiguration.Reply